Formula/rclone.rb

```diff
@@ -13,19 +13,25 @@ class Rclone < Formula
     sha256 "cea1a5bf6e0346731ba8357e313ae6eb3633e14400c2e9e67bd5a3f8524721f6" => :high_sierra
   end
 
-  deprecate! because: "requires FUSE"
-
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-tags", "cmount", *std_go_args
+    system "go", "build", "-tags", "brew", *std_go_args
```

Comment:
What does the `brew` tag do?

Reply:
This replaces the regular mount subcommand with a special error:

```
2020/11/17 22:34:05 Fatal error: failed to mount FUSE fs: mount is not supported on MacOS when installed via Homebrew. Please install the binaries available at https://rclone.org/downloads/ instead if you want to use the mount command
```

Reply:
This was added specially for Homebrew so I am open to changing the wording if you would like something different.